cocos/scripting/js-bindings/script/jsb_opengl_constants.js

```diff
@@ -826,6 +826,11 @@ gl.WRITEONLY_RENDERING_QCOM	= 0x8823;
 gl.WRITE_ONLY_OES	= 0x88b9;
 gl.Z400_BINARY_AMD	= 0x8740;
 gl.ZERO	= 0x0;
+gl.FRONT = 0x0404;
+gl.BACK = 0x0405;
```

Comment:
gl.FRONT, gl.BACK and gl.FRONT_AND_BACK already exist, please remove them.

And the list is ordered by alphabetic order, so please respect the order